packages/optimizer-docker/index.js

```diff
@@ -45,6 +51,11 @@ const port = 3000;
 if (process.env.NODE_ENV !== 'test') {
   server.listen(port);
   console.log(`AMP Optimizer listening at http://localhost:${port}`);
+  if (configuration.profile === true) {
```

Comment:
```suggestion
  if (configuration.profile) {
```

Reply:
this is always terrific for me to let javascript determine if my boolean is true :sweat_smile: